client/src/js/app.js

```diff
@@ -326,6 +325,26 @@ function bhimaConfig($stateProvider, $urlRouterProvider, $urlMatcherFactoryProvi
     templateUrl : 'partials/purchases/create/create.html'
   })
 
+  /* cashflow page */
+  .state('cashflow', {
+    url : '/finance/cashflow',
+    params: { dateFrom: null, dateTo: null, cashbox: null },
+    controller : 'CashflowController as IndexCtrl',
```

Comment:
Since you are dealing with multiple scopes/states, it's a good idea to be explicit in your names.  To avoid future confusion, I recommend you name this `CashflowIndexCtrl` or `CashflowCtrl`.